plugins/system/languagefilter/languagefilter.php

```diff
@@ -960,4 +965,295 @@ private function getLanguageCookie()
 
 		return $languageCode;
 	}
+
+	// Events for Default Association Language:
+
+	/**
+	 * Before Saving extensions
+	 * Method is called when an extension is going to be saved.
+	 * Change parameters for default association language as they depends on other parameters.
+	 *
+	 * @param   string  $context  The extension
+	 * @param   JTable  $table    DataBase Table object
+	 *
+	 * @return  void
+	 *
+	 * @since   4.0.0
+	 */
+	public function onExtensionBeforeSave($context, $table)
+	{
+		// Get the parameters which are to be saved.
+		$params          = json_decode($table->params);
+		$tableElement    = $table->element;
+		$pluginStatus    = $table->enabled;
+		$itemAssocStatus = $params->item_associations;
```

Comment:
```suggestion
		$itemAssocStatus = $params->item_associations ?? false;
```

Fix PHP Notice: Undefined property: stdClass::$item_associations in /plugins/system/languagefilter/languagefilter.php on line 989

Reply:
@lavipr Ignore this suggestion, it should be made different, like @SharkyKZ suggests.

Reply:
ah too late. well it does not do harm.

Reply:
Thank you for testing and helping :)